client/connection/ui.ts

```diff
@@ -2,17 +2,23 @@ import lang from "../lang"
 import { syncStatus } from "./state"
 import { handlers, message } from "./messages"
 
+type syncNum = {
+	clients: number
```

Comment:
Honestly, do we need to pass both numbers? Maybe just the active count is enough.  I'd prefer to declutter the UI. Thoughts?

Reply:
Well, it wouldn't be the skeleton counter without the idle number.
It looks fine when I tested it. If it's all that bad, we can remove it later.

Reply:
I'm thinking if only showing the active poster counter is a better idea. Minimalism and all that.

Reply:
Well, it's your decision ultimately, but I'd like to keep it.
I find it doesn't clutter things at least enough to matter, and provides useful information to users.